cache/cache_test.cc

```diff
@@ -686,6 +686,13 @@ TEST_P(CacheTest, DefaultShardBits) {
   ASSERT_EQ(6, sc->GetNumShardBits());
 }
 
+TEST_P(CacheTest, GetCharge) {
+  Insert(1, 2);
+  Cache::Handle* h1 = cache_->Lookup(EncodeKey(1));
+  ASSERT_EQ(2, DecodeValue(cache_->Value(h1)));
+  ASSERT_GT(cache_->GetCharge(h1), 0);
```

Comment:
Why not  `ASSERT_EQ(1, ache_->GetCharge(h1));`?

Reply:
Yes, it can be that. I have updated the commit.